app/[locale]/people/helper/PeopleCell.tsx

```diff
@@ -24,40 +24,41 @@ export default function PeopleCell({
   content,
 }: PeopleCellProps) {
   return (
-    <article className="group flex w-fit flex-row gap-5 text-md sm:w-36 sm:flex-col sm:gap-3">
-      <Link
-        href={href}
-        className="relative h-48 w-36"
-        style={{ filter: 'drop-shadow(0px 0px 4px rgba(0,0,0,0.15))' }}
-      >
-        <Image src={imageURL} alt="프로필 사진" className="object-cover" fill quality={25} />
-      </Link>
-      <div className="flex flex-col items-start break-keep">
+    <Link href={href}>
+      <article className="group flex w-fit cursor-pointer flex-row gap-5 text-md sm:w-36 sm:flex-col sm:gap-3">
```

Comment:
이유는 모르겠는데 Link 넣으니까 아래 에러 떠서 너도 그런지 확인좀!

Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Expected server HTML to contain a matching <article> in <a>.